src/lib/mdns/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -209,36 +216,80 @@ CHIP_ERROR AdvertiserMinMdns::Start(chip::Inet::InetLayer * inetLayer, uint16_t
 /// Stops the advertiser.
 CHIP_ERROR AdvertiserMinMdns::StopPublishDevice()
 {
-    mQueryResponderAllocatorOperational.Clear();
+    for (size_t i = 0; i < kMaxOperationalNetworks; ++i)
+    {
+        mQueryResponderAllocatorOperational[i].Clear();
+    }
     mQueryResponderAllocatorCommissionable.Clear();
     mQueryResponderAllocatorCommissioner.Clear();
     return CHIP_NO_ERROR;
 }
 
+int AdvertiserMinMdns::FindOperationalAllocator(const FullQName & qname) const
+{
+    for (size_t i = 0; i < kMaxOperationalNetworks; ++i)
+    {
+        if (mQueryResponderAllocatorOperational[i].GetResponder(QType::SRV, qname) != nullptr)
+        {
+            return i;
+        }
+    }
+    return -1;
+}
```

Comment:
This would probably be more typesafe if it returned `QueryResponderAllocator<kMaxOperationalRecords> *` (which could use a nicer typedef/using name) like so:
```suggestion
QueryResponderAllocator<kMaxOperationalRecords> * AdvertiserMinMdns::FindOperationalAllocator(const FullQName & qname) const
{
    for (auto & alloc : mQueryResponderAllocatorOperational)
    {
        if (alloc.GetResponder(QType::SRV, qname) != nullptr)
        {
            return &alloc;
        }
    }
    return nullptr;
}
```
and then the caller knows it has a `QueryResponderAllocator<kMaxOperationalRecords>` and what to do with it as opposed to needing to know what to do with the integer.

Reply:
That's fine too - did it as an index originally so I could make them const functions.